@xen-orchestra/backups/_deltaVm.js

```diff
@@ -16,6 +16,16 @@ exports.TAG_BASE_DELTA = TAG_BASE_DELTA
 const TAG_COPY_SRC = 'xo:copy_of'
 exports.TAG_COPY_SRC = TAG_COPY_SRC
 
+const CACHE = new Map()
+const resolveUuid = async (xapi, uuid, type) => {
+  let ref = CACHE.get(uuid)
+  if (ref === undefined) {
+    ref = await xapi.call(`${type}.get_by_uuid`, uuid)
+    CACHE.set(uuid, ref)
+  }
+  return ref
+}
```

Comment:
Be careful of the lifetime of this cache, maybe you should move it into `exportDeltaVm` and pass it as a an optional param.

Example: https://github.com/vatesfr/xen-orchestra/blob/6e6886a6bac6d8672c775991eebb00c991e54cfc/%40xen-orchestra/xapi/src/vm.js#L126